sebs/local/function.py

```diff
@@ -53,17 +54,21 @@ def __init__(
         self._instance.reload()
         networks = self._instance.attrs["NetworkSettings"]["Networks"]
         self._port = port
-        self._url = "{IPAddress}:{Port}".format(
-            IPAddress=networks["bridge"]["IPAddress"], Port=port
-        )
-        if not self._url:
-            self.logging.error(
-                f"Couldn't read the IP address of container from attributes "
-                f"{json.dumps(self._instance.attrs, indent=2)}"
-            )
-            raise RuntimeError(
-                f"Incorrect detection of IP address for container with id {self._instance_id}"
+
+        if is_linux():
```

Comment:
In the other PR, you added the if condition to storage and also checked if SeBS is running under WSL. Do we need that here as well?